src/MoBi.Core/Services/BuildingBlockVersionUpdater.cs

```diff
@@ -36,12 +36,17 @@ public void UpdateBuildingBlockVersion(IBuildingBlock buildingBlock, uint newVer
          if (buildingBlock == null)
             return;
 
-         if (shouldConvertToExtensionModule(buildingBlock, conversionOption)) 
+         if (shouldConvertToExtensionModule(buildingBlock, conversionOption))
             buildingBlock.Module.IsPKSimModule = false;
 
          if (shouldConvertToPKSimModule(buildingBlock, conversionOption))
             buildingBlock.Module.IsPKSimModule = true;
 
+         if (buildingBlock is IndividualBuildingBlock individual)
+            individual.SnapshotOriginModuleId = null;
+         else if (buildingBlock is ExpressionProfileBuildingBlock expressionProfileBuildingBlock)
+            expressionProfileBuildingBlock.SnapshotOriginModuleId = null;
+
```

Comment:
I wonder if for Individuals/ExprProfiles we need a similar revert logic as for Modules.
Maybe not right now but in one of the future PRs